_drafts/JSCON Budapest

```diff
@@ -0,0 +1,122 @@
+---
+
+title: 'JS CON  Budapest'
```

Comment:
Can we change this to 'JSConf Budapest'? As that is the actual title of the conference.